packages/frontend/model/extract-capi.ts

```diff
@@ -19,10 +19,10 @@ const apply = (input: string, ...fns: Array<(_: string) => string>): string => {
     return fns.reduce((acc, fn) => fn(acc), input);
 };
 
-const getEditionValue: (name: string) => Edition = name => {
+const getEditionValue: (name: string) => Edition | false = name => {
```

Comment:
Is false what we want over a default Edition?

Reply:
Yeah, just wanted the ability to distinguish when an edition is missing from a string.